include/s3select_oper.h

```diff
@@ -251,9 +251,16 @@ struct binop_pow {
     double operator()(double a,double b)
     {
         return pow(a,b);
-    }
+    } 
 };
 
+struct binop_modulo {
+    int operator()(int a,int b)
+    {
+        return a%b;
```

Comment:
its not correct to use int , should use int64_t.
should verify for legit operation (b!=0) , otherwise the hosting process(RGW) may crash.
the same should apply to binop_div.